NEWS.md

```
@@ -6,6 +6,12 @@ ClimaCoupler.jl Release Notes
 
 ### ClimaCoupler features
 
+#### Removed hierarchy experiments. PR[#1277](https://github.com/CliMA/ClimaCoupler.jl/pull/1277)
```

markdownlint-cli2 (0.17.2)

Heading style
Expected: setext; Actual: atx

(MD003, heading-style)